docs/user/dashboard/lens.asciidoc

```diff
@@ -303,3 +303,22 @@ Pagination in a data table is unsupported in *Lens*. However, the <<types-of-vis
 ===== Is it possible to have more than one y-axis scale in visualizations?
 
 *Lens* lets you pick, for each Y dimension, up to two distinct axis: *left* and *right*. Each axis can have a different scale.
+
+[float]
+[[why-my-value-is-not-of-the-right-color-using-value-based-coloring]]
+===== Why mt value is not of the right color when using value-based coloring?
+
+There could be various reason for a specific value in the table to have a different color than expected.
+
+Here's a short list of few different aspects to check:
+* Make sure the value falls within the desidered color stop value defined in the panel. Color stop values are "inclusive".
```

Comment:
I'm a little out of my realm here. Do users know what "inclusive" means when it comes to color stop values?

Reply:
Inclusivity, in this context refers to "ranges", so I would expect they are. I do not have a strong opinion here on the terminology, so if anything better comes I'm open.